### PR TITLE
feat: applicant details page content changes

### PR DESCRIPTION
**Pull request by n-r-b**
### JIRA

https://companieshouse.atlassian.net/browse/SR-131

### Change Description
Applicant Details page content changes.

Email address will be removed in separate ticket.

### Page Screenshot
Screenshot 2020-10-29 at 15 42 19

### Work Checklist

- [x] Tests added where applicable
- [x] Test coverage of new code meets target of 80%
- [x] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

---

### Merge Instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/validators/schema/ApplicantDetailsSchema.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import Joi from 'joi';
 import { YesNo } from '../../models/YesNo';
 import { basicString } from './BasicStringSchemaItem';
 
-const fullNameErrorMessage = 'Full name is required';
-const hasPreviousNameMissingMessage = 'Select yes if the applicant has used a different name for business purposes in the last 20 years';
+const fullNameErrorMessage = 'Enter the applicant’s full name';
+const hasPreviousNameMissingMessage = 'Select yes if the applicant has used a different name on the Companies house register in the last 20 years';
 const previousNameMissingMessage = 'Enter previous full names, used for business purposes';
```
Comment (rhysbarrett) [inline src/validators/schema/ApplicantDetailsSchema.ts:7]: Does the `previousNameMissing` message also need to be updated to bring it in line with the `hasPreviousNameMissing` message? The latter has moved away from "for business purposes" but the phrasing is still present in the former.

File: src/views/applicant-details.njk [hunk 2]
```diff
@@ -8,7 +8,7 @@
 {% from 'components/date-input.njk'                import dateInput %}
 
 {% block pageTitle %}
-  Applicant’s Details
+  {{ 'Applicant’s Details' | pageTitle(validationResult) }}
```
Comment (rhysbarrett) [inline src/views/applicant-details.njk:11]: The Changes PDF doesn't make it very clear but the page titles have been updated in V4 of the prototype.

```suggestion
  {{ 'What are the applicant’s details?' | pageTitle(validationResult) }}
```

File: src/validators/schema/ApplicantDetailsSchema.ts (hunk 1 above)
Comment (n-r-b) [reply src/validators/schema/ApplicantDetailsSchema.ts:7]: good point, i'll find out

File: src/views/applicant-details.njk (hunk 2 above)
Comment (n-r-b) [reply src/views/applicant-details.njk:11]: i'll also check this too as we've been told to follow the changes proposed in the PDF

**Review by rhysbarrett — APPROVED** [review]

File: src/validators/schema/ApplicantDetailsSchema.ts (hunk 1 above)
Comment (n-r-b) [reply src/validators/schema/ApplicantDetailsSchema.ts]: i've updated `previousNameMissing` message to `Enter full name`

File: src/views/applicant-details.njk (hunk 2 above)
Comment (n-r-b) [reply src/views/applicant-details.njk]: i've updated the error page title

**Pull request merged**